e2e-tests/version-service/conf/version-service-recommended-rs0.yml

```diff
@@ -5,6 +5,7 @@ metadata:
   finalizers:
     - delete-psmdb-pvc
 spec:
+  crVersion: 9.9.9
```

Comment:
we need to merge https://github.com/percona/percona-server-mongodb-operator/pull/988 and only after that improve/change this test if needed

Reply:
yep) I left comment for Ege about it.